web/modules/custom/dpl_favorites_list_material_component/dpl_favorites_list_material_component.links.menu.yml

```diff
@@ -0,0 +1,11 @@
+dpl_loan.list:
```

Comment:
Think there is some c&p here in the naming... `dpl_loan.list` ?

Reply:
fixed